packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4019,11 +4019,30 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   public usePhoneAudio(phoneNumber: string) {
```

Comment:
Yeah we can but do you find any advantage of using async await over .then?
One advantage is code readability but since it's a small function and we have mostly used .then in other methods so I have kept this function syntax in sync with other methods. Please let me know if I need to change it?

Reply:
Correct @meta-dipanshu-sharma, Sooner or later we might be doing that, So I thought to use that way, I am ok with correct implementation as well.